content/docs/faq-ajax.md

```diff
@@ -82,3 +82,50 @@ class MyComponent extends React.Component {
   }
 }
 ```
+
+Has the following equivalent with [Hooks](https://reactjs.org/docs/hooks-intro.html):
+
+```js
+function MyComponent() {
+  const [error, setError] = useState(null);
+  const [isLoaded, setIsLoaded] = useState(false);
+  const [items, setItems] = useState([]);
```

Comment:
This would be a good use case for a reducer (loading/error/hasItems are exclusive states). Thoughts on showing that here?

Reply:
I thought that as well at first. But going from a Class example using state, to a functional example using useReducer seems like a big jump. While useReducer is a better choice here in a real app, imo sticking with useState is better for this particular example.

Reply:
Maybe... they don’t need to be, though, so much as represent the recommended pattern for each

@sophiebits @gaearon thoughts?

Reply:
We'll go with this for now